File: ShadowFrameworkJS/src/shadow/pipeline/openGL20/SFGL20Pipeline.js

function SFGL20Pipeline_setup(){
	SFPipeline_setSfProgramBuilder(new SFGL20ProgramBuilder());
	SFPipeline_setSfPipelineGraphics(new SFGL20PipelineGraphics());
	SFPipeline_setSfPipelineMemory(new SFGL20PipelineMemory());
	SFPipeline_setSfTexturePipeline(new SFGL20TexturePipeline());
	SFExpressionGeneratorKeeper_getKeeper().setGenerator(new SFBasicExpressionGenerator());
}
	

File: ShadowFrameworkAndroid/src/com/shadow/android/opengles20/SFGL20Pipeline.java
package com.shadow.android.opengles20;

import shadow.pipeline.SFPipeline;
import shadow.pipeline.expression.SFBasicExpressionGenerator;
import shadow.pipeline.expression.SFExpressionGeneratorKeeper;
import shadow.pipeline.java.SFGL20PipelineMemory;

public class SFGL20Pipeline {
	public static void setup(){
		SFPipeline.setSfProgramBuilder(new SFGL20ProgramBuilder());
		SFPipeline.setSfPipelineGraphics(new SFGL20PipelineGraphics());
		SFPipeline.setSfPipelineMemory(new SFGL20PipelineMemory());
		SFPipeline.setSfTexturePipeline(new SFGL20TexturePipeline());
		SFExpressionGeneratorKeeper.getKeeper().setGenerator(new SFBasicExpressionGenerator());
	}
}
